UIKitMovie/UIKitMovieTests/UIKitMovieTests.swift

```diff
@@ -0,0 +1,67 @@
+// UIKitMovieTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftyJSON
+@testable import UIKitMovie
+import XCTest
+
+/// Unit-тест представления списка фильмов
+final class UIKitMovieTests: XCTestCase {
```

Comment:
```suggestion
final class MovieTests: XCTestCase {
```
не понятно по названию что тестируешь глянь у Даниила  пр